src/styles/blocks/gallery.scss

```
@@ -0,0 +1,97 @@
+.gallery {
+  padding: $standardPadding;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (max-width: 1000px) {
+    padding: $mobilePadding;
+  }
+
+  &__title {
+    font-size: 48px;
+    font-weight: bold;
+    text-align: center;
+
+    @media (max-width: 1000px) {
+      font-size: 36px;
+    }
+  }
+
+  &__crown {
+    margin-bottom: 37px;
+
+    @media (max-width: 1000px) {
+      margin-bottom: 29px;
+    }
+  }
+
+  &__wrapper-1 {
+    display: grid;
+    grid-template-columns: 4fr 2fr;
+    grid-template-rows: 256px;
+    gap: 20px;
+
+    @media (max-width: 1000px) {
+      grid-template-columns: 1fr 1fr;
+      grid-template-rows: 131px;
+    }
+  }
+
+  &__wrapper-2 {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 256px;
+    gap: 20px;
+
+    @media (max-width: 1000px) {
+      grid-template-rows: 131px;
+    }
+  }
```

```suggestion
  &__wrapper {
    display: grid;
    grid-template-columns: 1fr 1fr;
    grid-template-rows: 256px;
    gap: 20px;

    @media (max-width: 1000px) {
      grid-template-rows: 131px;
    }
  }
```

Let's make it common for all wrappers